addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -96,15 +106,15 @@ options.registry.TableOfContent = options.Class.extend({
         }
         // We dispose the scrollSpy because the navbar will be updated.
         this._disposeScrollSpy();
-        const $nav = this.$target.find('.s_table_of_content_navbar');
-        $nav.empty();
+        navEl.innerHTML = '';
         _.each(headingsEls, el => {
             const $el = $(el);
             const id = 'table_of_content_heading_' + _.now() + '_' + _.uniqueId();
-            $('<a>').attr('href', "#" + id)
+            const visibilityId = $el.closest('[data-visibility-id]').data('visibility-id');
```

Comment:
Couldn't this match a parent outside the s_table_of_content?
On the other line in this PR, above, you do `headingEl.closest('section').getAttribute('data-visibility-id');` where `headingEl` is actually `$el` equivalent of the current line.
Why doing it two different ways? The other one where you check the section seems better, as it won't go outside of the table_of_content parents

Reply:
Hello @rdeodoo,
Yes, `headingEl.closest('section').getAttribute('data-visibility-id')` seems better, as it won't go outside of the `table_of_content` parents. It was just a mistake, but I have made the suggested changes. Thanks!